import { GlobalErrorException } from '../../core/error/global-error-exception';

export class DeleteUserException extends GlobalErrorException {
  constructor(cause: string, httpCode: number) {
    super('Erro ao remover usu√°rio!', cause, httpCode);
  }
}
